src/components/MewCheckbox/MewCheckbox.vue

```diff
@@ -35,11 +29,20 @@ export default {
       default: function() {
         return { title: '', url: ''};
       }
+    },
+    value: {
```

Comment:
hmm is this suppose to match the prop name isChecked ?

Reply:
oohh i think i get it now

Reply:
yeah. so `v-model` is the same as passing `:value="something" @value="something"` ie:
```
  <mew-checkbox v-model="something"></mew-checkbox>
  
  <mew-checkbox :value="something" @input="something = $event"></mew-checkbox>
```

So it should work the same but when the value changes, the parent can react to it aswell.